src/coreclr/gc/unix/gcenv.unix.cpp

```diff
@@ -1170,7 +1170,7 @@ static size_t GetCurrentVirtualMemorySize()
     char* line = nullptr;
 
     // process virtual memory size is reported in the first column of the /proc/self/statm
-    FILE* file = fopen("/proc/self/statm", "r");
+    FILE* file = fopen("/proc/self/statm", "rb");
```

Comment:
This file doesn't need to be modified, it is compiled against system headers, not PAL.